fcrepo-auth-common/src/main/java/org/fcrepo/auth/common/ContainerRolesPrincipalProvider.java

```diff
@@ -42,11 +42,17 @@
 
     private static final Logger LOGGER = LoggerFactory.getLogger(ContainerRolesPrincipalProvider.class);
 
-    protected static class ContainerRolesPrincipal implements Principal {
+    /**
+     * @author Kevin S. Clarke
+     */
+    public static class ContainerRolesPrincipal implements Principal {
 
         private final String name;
 
-        ContainerRolesPrincipal(final String name) {
+        /**
+         * @param name
```

Comment:
[WARNING] /home/awoods/programming/java/apps/fedora/futures/fcrepo4/fcrepo-auth-common/src/main/java/org/fcrepo/auth/common/ContainerRolesPrincipalProvider.java:53: warning: no description for @param
[WARNING] * @param name